src/main/resources/nubesgen/terraform/main.tf.mustache

```diff
@@ -22,6 +22,16 @@ locals {
   // If an environment is set up (dev, test, prod...), it is used in the application name
   environment = var.environment == "" ? "dev" : var.environment
 }
+{{#AllowingClientIpRequired}}
+
+data "http" "myip" {
+  url = "http://ipv4.icanhazip.com"
```

Comment:
If no internal service available maybe it is possible to expose an endpoint at nubesgen.com that returns the client IP. something like:
https://nubesgen.com/myip

Reply:
@felipmiguel  After some digging, it seems we (Microsoft) can use http://whatismyip.akamai.com/.
Also there is http://whatismyip.akamai.com/advanced for more details (not useful in this context, but useful to know)

Reply:
Maybe it is possible to use deployment scripts in bicep to get the IP. https://docs.microsoft.com/en-us/azure/azure-resource-manager/bicep/deployment-script-bicep#sample-bicep-files

Reply:
I doubt this will work. Deployment scripts are run on a remote container, not on the deployment agent itself, so we'll not get the right IP address :/

Reply:
In this case for bicep I only see executing a tool out of the deployment script and passing the IP as a parameter.